app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,21 @@
+module API
+  module V1
+    class BooksController < ApplicationController
+      include Wor::Paginate
+
+      before_action :authenticate_user!
+
+      def index
+        books = Book.all
+
+        render_paginated books, serializer: Books::IndexSerializer
```

Comment:
Ummm  i think that here could be better to just use `render_paginated Book, serializer: Books::IndexSerializer` instead, because, the `.all` loads in memory all the records, in the case when you will have thousands of records, it will slow the request time.

Reply:
ok! done!